docs/configuration/openconfig.md

```diff
@@ -183,10 +182,50 @@ Starting in EOS `4.25.1F` it is possible to enable these mappings, for IPV4, IPV
 
 ```text
 management api models
-    ipv4-unicast
-    ipv6-unicast
+   ipv4-unicast
+   ipv6-unicast
+```
+
+## Telemetry Timestamps
+
+Per the GNMI specification, the default timestamp field of a notification message is set to be the time at which the
+value of the underlying data source changes or when the reported event takes place.  In order to facilitate integration
+in legacy environments oriented around polling style operations, an option to support overriding the timestamp field to
+the send-time is available. (as of 4.27.0F)
+
+Overriding the timestamp to `send-time` is applicable to all STREAM and POLL subscriptions.
+
+Configuration is outlined below.
+
+```text
+management api gnmi
+   transport grpc default
+      notification timestamp send-time
```

Comment:
should we mention that we can change it back to the defaults either by doing
`no notification timestamp send-time`
or `notification timestamp last-change-time` ?

Reply:
good idea.  i'll update.